wdl/transforms/new-base/src/main/scala/wdl/transforms/base/ast2wdlom/AstToCommandSectionElement.scala

```diff
@@ -10,9 +11,23 @@ object AstToCommandSectionElement {
 
     ast.getAttributeAsVector[CommandPartElement]("parts") map { parts =>
       val lines = makeLines(parts)
-      val trimmed = trimStartAndEnd(lines)
-      val commonPrefixLength = minimumStartLength(trimmed)
-      val commonPrefix = " " * commonPrefixLength
+      val trimmed = trimStartAndEndBlankLines(lines)
+
+      // Commands support either tabs or spaces. For the undefined case of mixed tabs and spaces, Cromwell returns an error.
```

Comment:
Why not a warning as per the spec's suggestion?

Reply:
It could be done, but the `stripStarts` function would have to be reworked quite a bit because it currently assumes uniform leading whitespace

I just stopped here because it seemed like the right point along the value <-> effort continuum